runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -341,19 +311,19 @@ impl API for Module {
         let account = storage::TypedStore::new(storage::PrefixStore::new(balances, &address));
 
         Ok(types::AccountBalances {
-            balances: BTreeMap::from_iter(account.iter()),
+            balances: account.iter().collect(),
         })
     }
 }
 
 impl Module {
     fn tx_transfer(ctx: &mut TxContext<'_, '_>, body: types::Transfer) -> Result<(), Error> {
-        // Reject transfers when they are disabled.
```

Comment:
Where did this check go? Also we should add a test for disabled transfers.

Reply:
fixed and added a test